config.go

```diff
@@ -50,14 +50,21 @@ func DefaultConfig(authToken string) ClientConfig {
 	}
 }
 
-func DefaultAzureConfig(apiKey, baseURL, engine string) ClientConfig {
+func DefaultAzureConfig(apiKey, baseURL string, modelMapper map[string]string) ClientConfig {
+	if len(modelMapper) == 0 {
+		modelMapper = map[string]string{
+			GPT3Dot5Turbo0301: "gpt-35-turbo-0301",
```

Comment:
What if `modelMapper` would be just a function that user can provide?

Reply:
Your suggestion is great, I tried to make those changes, the code become more concise and has better extensibility. I will submit the new code later.